app/helpers/passengers_helper.rb

```diff
@@ -2,8 +2,10 @@ module PassengersHelper
   def expiration_check(passenger)
     if passenger.will_expire_within_a_week?
       'will_expire_soon'
-    elsif passenger.expired_within_3_days?
-      'expired_3_days'
+    elsif passenger.expired_within_grace_period?
+      'expired_within_grace_period'
+    elsif passenger.expired_for_3_days?
+      'expired_for_3_days'
```

Comment:
Actually, @mboneil10 has a good point. If someday ST comes to us and says "hey, we want the grace period to be a week" how easy will it be to change? I'm thinking this could be a model constant somewhere. That way we'd only have to change it once.